src/components/LanguageDataGrid/LanguageColumns.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  DataGridProProps as DataGridProps,
+  GridColDef,
+  GridToolbarColumnsButton,
+  GridToolbarFilterButton,
+} from '@mui/x-data-grid-pro';
+import {
+  booleanColumn,
+  getInitialVisibility,
+  QuickFilterButton,
+  QuickFilterResetButton,
+  QuickFilters,
+  textColumn,
+  Toolbar,
+  useFilterToggle,
+} from '../Grid';
+import { LanguageNameColumn } from '../Grid/Columns/LanguageNameColumn';
+import { SensitivityColumn } from '../Grid/Columns/SensitivityColumn';
+import { LanguageDataGridRowFragment as Language } from './languageDataGridRow.graphql';
+
+export const LanguageColumns: Array<GridColDef<Language>> = [
+  LanguageNameColumn({
+    field: 'name',
+    headerName: 'Name',
+    valueGetter: (_, language) => language,
+  }),
+  {
+    field: 'ethnologue.code',
+    headerName: 'Eth Code',
+    ...textColumn(),
+    valueGetter: (_, { ethnologue }) => ethnologue.code.value,
+    width: 90,
+  },
+  {
+    field: 'registryOfLanguageVarietiesCode',
+    headerName: 'ROLV',
+    ...textColumn(),
+    width: 90,
+    valueGetter: (_, { registryOfLanguageVarietiesCode }) =>
+      registryOfLanguageVarietiesCode.value,
+  },
+  SensitivityColumn({}),
+  {
+    field: 'population',
+    headerName: 'Population',
+    type: 'number',
+    width: 100,
+    valueGetter: (_, { population }) => population.value,
+    filterable: false,
+  },
+  {
+    field: 'id',
+    headerName: 'ID',
+    valueGetter: (_, language) => language.id,
+    width: 150,
+  },
```

Comment:
I think it's fine to have this ID column here for identification, but I'm wondering if it should just be hidden by default. I think a lot of data points we have aren't important enough to take up space here, but could be needed in certain situations. So I think it's fine to have users turn on the column. Though obviously this could use some feedback. And whether or not they know how to do that at all is something different entirely, since it applies to all grids.